### PR TITLE
Balance monitoring + some super minor cleaning (1)

### PR DESCRIPTION
**Pull request by johnstonematt**

### CODE REVIEW

File: cmd/solana_exporter/exporter_test.go [hunk 1]
```diff
@@ -128,22 +130,22 @@ func (c *staticRPCClient) GetVersion(ctx context.Context) (string, error) {
...
-	lastSlot *int64,
+	ctx context.Context, firstSlot *int64, lastSlot *int64,
 ) (*rpc.BlockProduction, error) {
 	return &staticBlockProduction, nil
 }
 
+//goland:noinspection GoUnusedParameter
+func (c *staticRPCClient) GetBalance(ctx context.Context, address string) (float64, error) {
+	return balances[address], nil
+}
```
Comment (SEJeff) [inline cmd/solana_exporter/exporter_test.go:147]: If you used named returns, and just returned the named return here, you'd not need the linter config.

File: pkg/rpc/client.go [hunk 2]
```diff
@@ -201,15 +179,18 @@ func (c *Client) GetBlockProduction(ctx context.Context, firstSlot *int64, lastS
...
 	}
 	return &production, nil
 }
+
+func (c *Client) GetBalance(ctx context.Context, address string) (float64, error) {
+	var resp response[BalanceResult]
+	if err := c.getResponse(ctx, "getBalance", []interface{}{address}, &resp); err != nil {
+		return 0, err
+	}
+	return float64(resp.Result.Value / 1_000_000_000), nil
```
Comment (SEJeff) [inline pkg/rpc/client.go:195]: nit: this [magic number ](https://en.wikipedia.org/wiki/Magic_number_(programming)) for the lamport to SOL converstion should be a const.

**Review by SEJeff — APPROVED** [review]
We should probably update the test fixtures to reflect what a newer (1.18.x or soon to be 2.x) solana instance shows vs the 🧓  1.16.x.

File: pkg/rpc/client.go (hunk 2 above)
Comment (johnstonematt) [reply pkg/rpc/client.go:195]: fair nit, implemented soon

File: cmd/solana_exporter/exporter_test.go (hunk 1 above)
Comment (johnstonematt) [reply cmd/solana_exporter/exporter_test.go:147]: I would still get a warning that `ctx` is unused, or am I not understanding correctly?

**Pull request merged**